src/OutputWriters/windowed_time_average.jl

```diff
@@ -83,18 +84,59 @@ JLD2OutputWriter scheduled on TimeInterval(4 days):
 """
 function AveragedTimeInterval(interval; window=interval, stride=1)
     window > interval && throw(ArgumentError("Averaging window $window is greater than the output interval $interval."))
-    return AveragedTimeInterval(Float64(interval), Float64(window), stride, 0.0, false)
+    return AveragedTimeInterval(Float64(interval), Float64(window), stride, 0.0, 0, false)
 end
 
+# function initialize_schedule!(sch::AveragedTimeInterval, first_actuation_time::Number)
+#     sch.first_actuation_time = first_actuation_time
+#     sch.actuations = 0
+#     return true
+# end
+
+# initialize_schedule!(sch::AveragedTimeInterval, model) = initialize_schedule!(sch, model.clock.time)
+
+
+function next_actuation_time(sch::AveragedTimeInterval)
+    t₀ = sch.first_actuation_time
+    N = sch.actuations
+    T = sch.interval
+    return t₀ + (N + 1) * T
+end
```

Comment:
This look good, but requires that `sch.actuations` are being incremented (by `sch.actuations +=1`) at the right times. I think that is the problem right now.